### ISSUE TITLE
How to run different tasks on multiple servers

### ISSUE THREAD

**Issue by mikegleasonjr** [shipitjs/shipit#60]
In staging I have 3 servers: 2 frontends and a load balancer.

The set of tasks to execute are different on the frontends and the load balancer.

I believe that only one set of servers can be specified and the tasks will act on them.

How can I set up the tasks to work on different set of servers at the same time without running shipit 2 times in a row (by specifying for example a deploy-staging-frontend and deploy-staging-loadbalancer tasks)

OR by doing something more hacky like:

```
// do things on frontend servers
// [...]
shipit.config.servers = ['loadbalancer.com']
shipit.initSshPool();
// [...]
// now do things on the load balancer
```

TL;DR;
How can I specify multiple pools and run tasks on specific pools?

Thank you

**Comment by neoziro** [comment]
@mikegleasonjr you can't for now.
